### DISCUSSION TITLE
cFprime and Cosmos for Telemetry tracking and Command Sequencing

### DISCUSSION THREAD

**Discussion by nchanilk** [ThibFrgsGmz/ThibFrgsGmz#4] [General]
Hi ThibFrgsGmz,
I'm Anil and exploring the fPrime GDS. After going through the link here: https://github.com/nasa/fprime/discussions/1269, I've found similarity between our work and would like to get in touch with you and brain storm my use case. Can you please help me with below?

1. Did you save the data into the database through fPrime for historical and telemetry tracking? In case yes, how dids you achieve and which DB you are using?
2. Did you integrate with Cosmos (We're thinking of itegrating with the Cosmos)/or any other MC UI?
3. I'm curuious to know how actually connecting to the Satelite dish works with the fprime-gds. 
Could you please provide any guidance on above? appreciate your help  and thanks in advance. 

Thanks,
Anil

**Comment by ThibFrgsGmz** [comment]
Hi Anil,

We haven't done anything yet; it was just gathering information to propose a demo project using the FPrime framework.
But I can answer some of your questions about what I am planning.

For the database model, maybe SQLite or Time Series Database which is widely used in IoT.

Personally, I'm not concerned with connecting to the satellite dish.
Indeed, I am only looking to provide an instrument to a satellite manufacturer; my customer is the satellite on-board computer.

However, be aware that the fprime-gds is a simple gse for the developer to help him in his development.

It is not intended to be used as a production tool or in control centers.

Right, @LeStarch?

**Reply by LeStarch** [reply]
Yes, the fprime GDS was built for in-lab development.  Hence the lack of a built-in database. Projects needing it to be more may adapt or replace it.

**Comment by nchanilk** [comment]
Thanks, @ThibFrgsGmz  for the information.


Thanks,
Anil
